cmd/operator-sdk/generate/openapi.go

```diff
@@ -64,7 +64,7 @@ func openAPIFunc(cmd *cobra.Command, args []string) error {
       which ./bin/openapi-gen > /dev/null || go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
 
       # Run openapi-gen for each of your API group/version packages
-      ./bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/<group>/<version> -O zz_generated.openapi -p ./pkg/apis/<group>/<version> -h ./hack/boilerplate.go.txt -r "-"
+      ./bin/openapi-gen -o "" -i ./pkg/apis/<group>/<version> -O zz_generated.openapi -p ./pkg/apis/<group>/<version> -h ./hack/boilerplate.go.txt -r "-"
```

Comment:
Required to keep the command `operator-sdk generate k8s` working.

Reply:
https://github.com/operator-framework/operator-sdk/pull/2740

Reply:
Hi @joelanford,

Really tks for the input. See that the flag in the version used is set as True by default. 
See: https://github.com/kubernetes/kube-openapi/blob/d50a959ae76a85c7c262a9767ef29f37093c2b8a/cmd/openapi-gen/openapi-gen.go#L41